<launch>
    
  <include file="$(find tfmini_ros_tiers)/launch/tfmini.launch" />

  <node pkg="dwm1001_uwb_tag_drivers" type="dwm1001_active" name="dwm1001_active" output="screen">
    <param name="port" type="string" value="/dev/ttyACM1" />
    <param name="tag_name" type="string" value="dronie" />
  </node>

  <node pkg="offboard_control" type="uwb_lidar_vio_to_mavros" name="uwb_lidar_vio_to_mavros" output="screen">
    <param name="uwb_topic" type="string" value="/dwm1001/tag/dronie/position" />
    <param name="vio_topic" type="string" value="/uav/camera/odom/sample" />
    <param name="lidar_topic" type="string" value="/uav/tfmini_ros_node/range" />
    <param name="output_topic" type="string" value="/uav/mavros/vision_pose/pose" />
  </node>

  <include file="$(find mavros)/launch/px4.launch" />

  <!--  
  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <arg name="camera"     value="t265" />
    <arg name="serial_no"  value="947122110535" />
  </include>
  <include file="$(find realsense2_camera)/launch/rs_camera.launch">
    <arg name="camera"     value="d435" /> 
    <arg name="serial_no"  value="938422071231" />
    <arg name="filters"    value="pointcloud" />
  </include>
  -->

</launch>
